agents/ppo.py

```diff
@@ -23,17 +24,22 @@ def __init__(
         assert 0 < discount_factor < 1, 'Incorrect `discount_factor` choice'
         assert 0 < ratio_clip < 1, 'Incorrect `ratio_clip` choice'
         super().__init__()
+        self._prefix = prefix
+        self._out_prefix = out_prefix
         self._actor = actor
         self._critic = critic
         self._discount_factor = discount_factor
         self._ratio_clip = ratio_clip
 
         self._actor_loss_weight = actor_loss_weight
         self._critic_loss_weight = critic_loss_weight
+        self._memory = []
```

Comment:
As far as I remember we decided to have shared static memory and pass in constructor as a param